### PR TITLE
Add Link to Legacy Page

### PR DESCRIPTION
**Pull request by kunalbotla**
The Legacy "G Suite" page is now hidden on Bank but is still a needed page for deletion functionality. This PR adds a direct link to the legacy page for each Google Workspace from their current page.

### CODE REVIEW

File: app/models/g_suite.rb [hunk 1]
```diff
@@ -66,6 +66,10 @@ def dns_check_url
     "https://nslookup.io/dns-records/#{domain}"
   end
 
+  def legacy_page_url
+    "https://bank.hackclub.com/g_suites/#{id}"
```
Comment (maxwofford) [inline app/models/g_suite.rb]: This should probably be a rails url helper to still work correctly in development/preview

**Comment by kunalbotla** [comment]
oh I like this, but I think we should also link to the old page so we can view it if we need to.

**Comment by exu3** [comment]
👍, Is it cool if I commit those changes from the screen recording here?

**Comment by kunalbotla** [comment]
for sure! i'll make a few phrasing edits before final review but it should be mostly set from there

**Review by maxwofford — APPROVED** [review]
LGTM! I'm cool with this getting merged whenever you're ready

**Comment by exu3** [comment]
@kunalbotla is this good to merge?

**Comment by kunalbotla** [comment]
yep! merging

**Pull request merged**